README.md

```diff
@@ -74,6 +74,7 @@ Please see the Examples section for more details.
 
 * xHyperV: Added SecureBoot parameter to enable control of the secure boot BIOS setting on generation 2 VMs.
 -  Fixed drive letter when mounting VHD when calling resource xVhdFile. Fixes #20.
+* xVMHyperV: Added enabling of Guest Service Interface.
 
 ### 3.2.0.0
```

Comment:
Add an  section for an 'unreleased' version and summarize the changes for that version.